tests/src/com/unciv/logic/map/UnitMovementTests.kt

```diff
@@ -199,7 +199,7 @@ class UnitMovementTests {
 
         city.hasJustBeenConquered = true
         civInfo.diplomacy[otherCiv.civName] = DiplomacyManager(otherCiv, otherCiv.civName)
-        civInfo.getDiplomacyManager(otherCiv).diplomaticStatus = DiplomaticStatus.War
+        civInfo.getDiplomacyManager(otherCiv).diplomaticStatus == DiplomaticStatus.War
```

Comment:
Sounds like a mistake

Reply:
Good catch! This was from when I reverted isAtWar() back.